soc/nxp/imx/imx9/imx943/Kconfig.defconfig.mimx94398.m33

```diff
@@ -30,4 +30,7 @@ config ETH_NXP_IMX_MSGINTR
 config CACHE_MANAGEMENT
 	default y
 
+config SCMI_CHAN_SEM_TIMEOUT_USEC
```

Comment:
The commit message says: "to fix a bug that failed to setup pins" - this is too generic. 
Please provide more details and context. It appears to be a hardware-related issue, with additional steps required for the i.MX943?
How did you identify the bug?
Why is a delay 100 times longer than the default necessary?